gnd/src/test/java/com/google/android/gnd/persistence/local/LocalDataStoreTest.java

```diff
@@ -67,97 +66,138 @@
+          .setUrl("some_url 2")
+          .build();
+
+  private static final Tile TEST_TILE_FAILED =
+      Tile.newBuilder()
+          .setId("id_3")
+          .setState(State.FAILED)
+          .setPath("some_path 3")
+          .setUrl("some_url 3")
+          .build();
+
+  private static final OfflineArea TEST_OFFLINE_AREA =
+      OfflineArea.newBuilder()
+          .setId("id_1")
+          .setBounds(LatLngBounds.builder().include(new LatLng(0.0, 0.0)).build())
+          .setState(OfflineArea.State.PENDING)
+          .build();
+
   // This rule makes sure that Room executes all the database operations instantly.
   @Rule public InstantTaskExecutorRule instantTaskExecutorRule = new InstantTaskExecutorRule();
 
   @Inject LocalDataStore localDataStore;
 
-  private static User createTestUser() {
-    return User.builder()
-        .setId("test_user_id")
-        .setEmail("test@gmail.com")
-        .setDisplayName("test user")
-        .build();
-  }
-
-  private static Project createTestProject() {
-    Builder multipleChoiceBuilder =
-        MultipleChoice.newBuilder().setCardinality(Cardinality.SELECT_ONE);
-    multipleChoiceBuilder
-        .optionsBuilder()
-        .add(Option.newBuilder().setCode("a").setLabel("Name").build())
-        .add(Option.newBuilder().setCode("b").setLabel("Age").build());
-
-    Field field =
-        Field.newBuilder()
-            .setId("field id")
-            .setLabel("field label")
-            .setRequired(false)
-            .setType(Type.MULTIPLE_CHOICE)
-            .setMultipleChoice(multipleChoiceBuilder.build())
-            .build();
-
-    Element element = Element.ofField(field);
-
-    Form form =
-        Form.newBuilder()
-            .setId("form id")
-            .setElements(ImmutableList.<Element>builder().add(element).build())
-            .build();
-
-    Layer layer =
-        Layer.newBuilder()
-            .setId("layer id")
-            .setItemLabel("item label")
-            .setListHeading("heading title")
-            .setDefaultStyle(Style.builder().setColor("000").build())
-            .setForm(form)
-            .build();
-
-    Project.Builder builder =
-        Project.newBuilder()
-            .setId("project id")
-            .setTitle("project 1")
-            .setDescription("foo description");
-    builder.putLayer(layer.getId(), layer);
-    return builder.build();
-  }
-
-  private static FeatureMutation createTestFeatureMutation(
-      String userId, String projectId, String layerId) {
-    return FeatureMutation.builder()
-        .setId(1L)
-        .setType(Mutation.Type.CREATE)
-        .setUserId(userId)
-        .setProjectId(projectId)
-        .setFeatureId("feature id")
-        .setLayerId(layerId)
-        .setNewLocation(
-            Optional.ofNullable(Point.newBuilder().setLatitude(110.0).setLongitude(-23.1).build()))
-        .setClientTimestamp(new Date())
-        .build();
+  private static void assertObservationMutation(
```

Comment:
One asserts that something Xs, so here it would be `assertObservationMutationEquals`.

But this also does something extra, so should be reflected in the name, e.g.:
`assertEqualsIgnoreId`.

We can omit "ObservationMutation" from the name for brevity since it will be clear from the arguments what's being compared.